channeld/channeld.c

```diff
@@ -2886,7 +2926,6 @@ static void handle_funding_depth(struct peer *peer, const u8 *msg)
 		peer->short_channel_ids[LOCAL] = *scid;
 
 		if (!peer->funding_locked[LOCAL]) {
-
```

Comment:
This level of subtlety *requires* a comment!

Suggest: /* This line intentionally left blank */